media/img/aboutRights.xhtml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Wrong file, wrong place?

Reply:
yep, this shouldn't be in here at all, thank you